File: parent/pinyougou-web-manager/src/main/webapp/js/service/userService.js
//服务层
app.service('userService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../user/findAll.do');
	}

	//冻结
	this.freeze=function(ids){
		return $http.get('../user/freeze.do?ids='+ids);
	}
    this.unfreeze=function(ids){
        return $http.get('../user/unfreeze.do?ids='+ids);
    }
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../user/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: parent/pinyougou-interface/src/main/java/cn/itcast/core/service/UserService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.user.User;
import entity.PageResult;

import java.util.List;

public interface UserService {
    void sendCode(String phone);

    void add(User user, String smscode);

    List<User> findAll();

    PageResult search(Integer page, Integer rows, User user);

    void freeze(Long[] ids);

    void unfreeze(Long[] ids);


    void update(String username);
}
